catalogue/aarch64-VMSA/README.md

```diff
@@ -0,0 +1,72 @@
+aarch64-VMSA: Catalogue with the semantics of Armv8-A VMSA
+============================================================
+
+Includes tests that exercise the semantics of the Virtual Memory
+System Architecture. Most of the tests in this catalogue, concurrently
```

Comment:
I would drop this comma - also why "most of the tests"? not all of them?

Reply:
They are some which have static faults. For example LDRv0.